packages/mdc-textfield/icon/README.md

```diff
@@ -21,6 +21,18 @@ Icons describe the type of input a text field requires. They can also be interac
 
 ## Basic Usage
 
+### Icon Set
```

Comment:
This doesn't belong here. If anything, make it a subheading under HTML Structure.